packages/mosaic/core/src/Coordinator.ts

```diff
@@ -38,9 +45,9 @@ export function coordinator(
 export class Coordinator {
   public manager: QueryManager;
   public preaggregator: PreAggregator;
-  public clients?: Set<MosaicClient>;
-  public filterGroups?: Map<Selection, any>;
-  protected _logger?: Logger;
+  public clients: Set<MosaicClient> = new Set;
```

Comment:
I often write it like this which I find easier.

```suggestion
  public clients = new Set<MosaicClient>;
```